### PR TITLE
Database design

### LINKED ISSUE #59: Review and compile list of entities and database components from requirements
Go through the functional requirements document and create a list of any concept in the requirements that would belong in the database schema.

### LINKED ISSUE #60: Create ER diagram for database model

### PR DESCRIPTION
**Pull request by JadenBalogh**
Closes #59, closes #60

### CODE REVIEW

File: Documentation/Dev/Environment Setup.md [hunk 1]
```diff
@@ -13,16 +13,18 @@ If either of these ports is taken by something already message the group and we
 4. Setup your MySQL Root Password. 
 5. Setup a dev user, this is the account we should be using to test our development setup.
 
-NOTE: Having Usernames and Password in github IS VERY INSECURE, DO NOT EXPOSE YOUR DB TO THE INTERNET. This username and password is something I made up just for this.
+Note: Having Usernames and Password in github IS VERY INSECURE, DO NOT EXPOSE YOUR DB TO THE INTERNET. This username and password is something I made up just for this.
 ```cmd
 User-type: DB Admin
 Username: notAdmin
 Password: notAdmin99!!
 ```
```
Comment (Jaredhw) [inline Documentation/Dev/Environment Setup.md:21]: Needs to be updated to be pulled from a local setup file and not hard-coded.

File: Documentation/Dev/Environment Setup.md [hunk 2]
```diff
@@ -31,7 +33,7 @@ Then you will have to sign into your MySQL Server via CMD with:
 mysql -u root -p
```
Comment (Jaredhw) [inline Documentation/Dev/Environment Setup.md:33]: Update this to be clearer.

File: server/Node.js [hunk 3]
```diff
@@ -0,0 +1,171 @@
+var mysql = require('mysql');
+//create connection
+var mysql = require('mysql');
+
```
Comment (Jaredhw) [inline server/Node.js]: Duplicate, clean up.

**Review by Jaredhw — DISMISSED** [review]

**Review by aKrisnanto — DISMISSED** [review]
"Node.js" should be renamed, other files look good

**Review by AA666666 — APPROVED** [review]

**Comment by AA666666** [comment]
Well done

File: Documentation/Dev/Environment Setup.md (hunk 1 above)
Comment (JadenBalogh) [reply Documentation/Dev/Environment Setup.md:21]: Also this documentation should be updated when that happens to specify that any username and password are acceptable so long as the constants file is setup properly.

File: Documentation/Dev/Database Design.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
+# Database Design
+This is a summary of the different entities and relationships extracted from our functional requirements.
+I'd like to add the MySQL Datatypes to this but I'm not sure if that's overkill. 
+## Entities
```
Comment (JadenBalogh) [inline Documentation/Dev/Database Design.md:4]: This is a fairly extensive list, nice work. We'll stick to the requirements for this milestone when actually creating the database for now, though.

**Review by Jaredhw — APPROVED** [review]

**Pull request merged**